internal/fourslash/tests/gen/findAllRefsForDefaultExport03_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func TestFindAllRefsForDefaultExport03(t *testing.T) {
 	t.Parallel()
-	t.Skip()
+
```

Comment:
Hm, I think this is actually already fixed on main. I don't know why main CI is not catching this...

Reply:
#1922, sigh.